Harvest.Web/ClientApp/src/Requests/AccountsInput.tsx

```diff
@@ -148,6 +194,11 @@ export const AccountsInput = (props: Props) => {
         onChange={onSelect}
         options={searchResultAccounts}
       />
+      {usecoa === true && (
+        <button className="btn btn-primary" onClick={() => lookupcoa()}>
```

Comment:
just a style thing, `onClick={() => lookupcoa()}` can be written as `onClick={lookupcoa}` which i think is nicer to read